Modules/Microsoft365DSC/DSCResources/MSFT_ODSettings/MSFT_ODSettings.schema.mof

```diff
@@ -11,6 +11,7 @@ class MSFT_ODSettings : OMI_BaseResource
     [Write, Description("Lets administrators set policy on access requests and requests to share in OneDrive for Business"),ValueMap{"On","Off","Unspecified"},Values{"On","Off","Unspecified"}] String ODBAccessRequests;
     [Write, Description("Block sync client on Mac")] Boolean BlockMacSync;
     [Write, Description("Disable dialog box")] Boolean DisableReportProblemDialog;
+    [Write, Description("enable/disable Safe domain List - if disabled overrides DomainGuids value")] Boolean TenantRestrictionEnabled;
```

Comment:
This drives documentation. Can we put a capital 'E' to Enable.

Reply:
No worries, I've amended that and also corrected a verbose message I'd altered while I was testing.